### PR TITLE
fix javaagent version

### PR DESCRIPTION
**Pull request by tormodatt**
**Description:** I noticed that the javaagent was running an old version.

**Link to tracking Issue(s):** <Issue number if applicable>

- Resolves: #issue-number

**Testing:** <Describe what testing was performed and which tests were added.>

**Documentation:** <Describe the documentation added.>

### CODE REVIEW

File: versions.txt [hunk 1]
```diff
@@ -15,7 +15,7 @@ operator-opamp-bridge=0.103.0
 
 # Represents the current release of Java instrumentation.
 # Should match autoinstrumentation/java/version.txt
```
Comment (tormodatt) [inline versions.txt:17]: it is already 2.5.0 here: https://github.com/open-telemetry/opentelemetry-operator/blob/main/autoinstrumentation/java/version.txt

Comment (pavolloffay) [reply versions.txt:17]: We are intentionally pinning 1.x due to breaking change on the semantic conventions

**Review by pavolloffay — CHANGES_REQUESTED** [review]

File: versions.txt (hunk 1 above)
Comment (tormodatt) [reply versions.txt:17]: right. is there any info about this and the plan for upgrade somewhere?

the comment should probably be updated in any case?

Comment (pavolloffay) [reply versions.txt:17]: It should be documented in release.yaml. 

Users can override the image in the instrumentation CR

Comment (tormodatt) [reply versions.txt:17]: aha, yes I can see that there is a comment about it in RELEASE.md. might help others if it were a bit easier to find though.

should I just close this PR then?

Comment (TylerHelmuth) [reply versions.txt:17]: There is also this pinned issue: https://github.com/open-telemetry/opentelemetry-operator/issues/2542

Comment (TylerHelmuth) [reply versions.txt:17]: @tormodatt closing is appropriate as we are not ready to accept making 2.x the default image.

**Pull request closed without merging**